pavelib/paver_tests/test_paver_quality.py

```diff
@@ -253,120 +225,3 @@ def test_report_dir_without_files(self):
         os.remove(self.test_file.name)
         pavelib.quality._prepare_report_dir(path(self.test_dir))  # pylint: disable=protected-access
         assert os.listdir(path(self.test_dir)) == []
-
-
-class TestPaverRunQuality(PaverTestCase):
-    """
-    For testing the paver run_quality task
-    """
-
-    def setUp(self):
-        super().setUp()
-
-        # mock the @needs decorator to skip it
-        patcher = patch('pavelib.quality.sh')
-        self._mock_paver_sh = patcher.start()
-        self.addCleanup(patcher.stop)
-
-        self.report_dir = tempfile.mkdtemp()
-        report_dir_patcher = patch('pavelib.utils.envs.Env.REPORT_DIR', path(self.report_dir))
-        report_dir_patcher.start()
-        self.addCleanup(shutil.rmtree, self.report_dir)
-        self.addCleanup(report_dir_patcher.stop)
-
-    @patch(OPEN_BUILTIN, mock_open())
-    def test_failure_on_diffquality_pylint(self):
-        """
```

Comment:
are there any other tests that cover the current implementation of how quality checks run? or we should refactor these tests to remove diff quality specific assertions

Reply:
After reviewing the tests again, I believe we don't need to modify these tests as these were only present to test the diff-quality section specifically. Other tests are already testing each quality check.